app/server/lib/FlexServer.ts

```diff
@@ -1987,6 +1997,35 @@ export class FlexServer implements GristServer {
     this.create.addExtraHomeEndpoints(this, this.app);
   }
 
+  public getLatestVersionAvailable() {
+    return this._latestVersionAvailable;
+  }
+
+  public setLatestVersionAvailable(latestVersionAvailable: LatestVersionAvailable): void {
+    log.info(`Setting ${latestVersionAvailable.version} as the latest available version`);
+    this._latestVersionAvailable = latestVersionAvailable;
+  }
+
+  public async publishLatestVersionAvailable(latestVersionAvailable: LatestVersionAvailable): Promise<void> {
+    log.info(`Publishing ${latestVersionAvailable.version} as the latest available version`);
+
+    if (process.env.REDIS_URL) {
+      const client = createClient(process.env.REDIS_URL);
```

Comment:
If we already have _redisSubscriptionClient, what is this second client here doing?

Reply:
On reading some docs - it looks like you need a dedicated client for subscriptions, I'm guessing that's why?